pyvista/plotting/render_window_interactor.py

```diff
@@ -1540,63 +1529,81 @@ def close(self):
         self._timer_event = None
 
 
-def _style_factory(klass):
-    """Create a subclass with capturing ability, return it."""
-    # We have to use a custom subclass for this because the default ones
-    # swallow the release events
-    # http://vtk.1045678.n5.nabble.com/Mouse-button-release-event-is-still-broken-in-VTK-6-0-0-td5724762.html
+class _CustomInteractorStyle:
```

Comment:
The functionality of `CustomInteractorStyle` is refactored from the old `CustomStyle` inside `_style_factory`